README.adoc

```diff
@@ -58,14 +58,17 @@ include::{common-includes}/gitclone.adoc[]
 
 include::{common-includes}/twyb-intro.adoc[]
 
-You can access the following microservices:
+The `system` microservice simulates a service that returns the system
+property information for its host. 
+The `system` service is accessible at the http://localhost:9080/system/properties[http://localhost:9080/system/properties^] URL. In this case, `localhost` is the host name.
 
-* The http://localhost:9080/system/properties[http://localhost:9080/system/properties^] microservice simulates the remote `system` service that retrieves the system property information for a specific host. In this case, `localhost` is a specific host name.
-
-* The http://localhost:9080/inventory/systems/localhost[http://localhost:9080/inventory/systems/localhost^] microservice is the `inventory` service that invokes the http://localhost:9080/system/properties[http://localhost:9080/system/properties^] microservice to retrieves the system property information.
-
-* The \http://localhost:9080/inventory/systems/{your_hostname} microservice is the `inventory` service that invokes the \http://{your_hostname}:9080/system/properties microservice. In Windows, Mac OS, and Linux, get your fully qualified domain name (FQDN) by entering `hostname` from your terminal. Visit the URL by replacing `{your_hostname}` with your FQDN.
-You will see the same system property information, but the process of getting the information is different.
+The `inventory` microservice makes a request to the `system` microservice and
+stores the system property information. 
+To fetch and store your system's information, enter the http://localhost:9080/inventory/systems/localhost[http://localhost:9080/inventory/systems/localhost^] URL into your browser. 
```

Comment:
To fetch and store your system's information
--->
To fetch and store your system information